pyproject.toml

```diff
@@ -628,8 +628,8 @@ evaluation = "max(0, 0 if fatal else 10.0 - ((float(5 * error + warning + refact
 # Tells whether to display a full report or only the messages.
 # reports =
 
-# Activate the evaluation score.
-score = true
+# Activate the evaluation ruleScore.
+ruleScore = true
```

Comment:
irrelevant change, revert

Reply:
No idea why this happened unless the IDE rename has made this change in a refactor error.